covalent/_shared_files/context_managers.py

```diff
@@ -113,15 +113,15 @@ def get_active_dispatch_info(self) -> "DispatchInfo":
         return self._active_dispatch_info
 
     @contextmanager
-    def claim(self, dispatch_info) -> None:
+    def claim(self, dispatch_info: "DispatchInfo") -> Generator:
         """
         Claims the given dispatch info as active.
 
         Args:
             dispatch_info: The dispatch info object to claim.
 
         Returns:
-            None
+            Returns a Generator object, that gets converted into a contextmanager.
```

Comment:
Minor (non-blocking suggestion): `Returns a generator, which gets converted into a context manager.`

Reply:
Sure, I'll update and let you know.